.circleci/config.yml

```diff
@@ -263,14 +263,15 @@ jobs:
       - run_test:
           pattern: "flake8"
 
-  build_base:
+  build_base_venvs:
     executor: ddtrace_dev
+    parallelism: 6
     steps:
       - checkout
-      - run: pip3 install riot
+      - run: pip install riot
       - run:
           name: "Generate base virtual environments."
-          command: "riot -v generate"
+          command: "echo '2.7,3.5,3.6,3.7,3.8,3.9' | tr ',' '\n' | circleci tests split | xargs -I PY riot -v generate --python=PY"
```

Comment:
I wonder if there is a better way than `echo '2.7,3.5,3.6,3.7,3.8,3.9'` will we have to repeat this in a ton of places? 

Is there anything we can add to riot to make it easier?

e.g.

```
riot list --python | circleci test split | riot -v generate
```

Not sure this example actually makes sense or not.

CircleCI also supports environment variables with test splitting, wonder if that is better way to support riot?

https://circleci.com/docs/2.0/parallelism-faster-jobs/#using-environment-variables-to-split-tests

e.g.

```
parallelism: 6
  steps
    - run: riot -v generate
```

and then `riot` can read from `CIRCLECI_NODE_INDEX` and `CIRCLECI_NODE_TOTAL` to break up the envs itself to build?

Reply:
> I wonder if there is a better way than echo '2.7,3.5,3.6,3.7,3.8,3.9' will we have to repeat this in a ton of places?

Only twice :)

> Is there anything we can add to riot to make it easier?

@majorgreys and I discussed adding built-in support for the test splitting (including the circleci env vars).